README.md

```diff
@@ -32,9 +34,25 @@ To run:
 
     docker run -it tiledb:release
 
-## Optional components
+### Optional components
 
 If you'd like to build TileDB with HDFS, use the `enable` build argument
 when building the images, e.g.:
 
     docker build --build-arg enable=hdfs -t tiledb:release
+
+## TileDB-R
+
+There is an image available for [TileDB-R](https://github.com/TileDB-Inc/TileDB-R)
+which is used only for CI. This image is build and designed to facilitate CI.
+
+### Instructions
+
+1. Install the Docker daemon from https://www.docker.com/community-edition
+
+2. Clone the TileDB-Docker repo and build the images:
+```
+git clone https://github.com/TileDB-Inc/TileDB-Docker
+cd TileDB-Docker
+docker build -t tiledb:ci-r ci-r
```

Comment:
If we build the container at hub.docker.com then pulling is easier than building so we could mention it.  But not that important.

Reply:
The README is build instructions for in general.

Reply:
For the CI build/deploy side we will eventually move this repo from travis to azure and setup azure for deploying to dockerhub. The end goal being to simplify and standardize our CI infrastructure to just Azure for our public repos.